### ISSUE TITLE
Incorporate ngrx/store

### ISSUE THREAD

**Issue by darenkeck** [Margorah/CharSheetNg2#5]
Use ngrx/store to manage state.

This includes:
* update package.json to incorporate ngrx/store
* write reducers to extract characters, stats
* create the set of actions for each of these data types
* might end up defining a model containing user data as well.

**Comment by darenkeck** [comment]
I have a bit more reading I need to do before we can get this incorporated. 

For now we'll take a different approach until we incorporate store, preparing for eventual integration. Move state delivery to a single service, have each component using that service and handling receiving state through observables. I will open a separate issue for the state service we are planning to create.

Closing this for now.
